sdk/confidentialledger/confidential-ledger-rest/CHANGELOG.md

```diff
@@ -7,8 +7,12 @@
 - User defined endpoint
 - User defined endpoint runtimeoptions
 - User defined endpoint modules
-- User defined roles 
-- 
+- User defined roles
```

Comment:
please merge the change log for both 1.1.2 and 1.1.0-beta.1 and remove 1.1.0-beta.1, we only need to keep one unreleased version.